client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/style.scss

```diff
@@ -139,7 +139,11 @@ $tab-background-color: #f4fbff;
 
 		.hosting-v2__nav-item-label {
 			color: var(--color-accent-90);
-			@include a4a-font-heading-md($font-weight: 600);
+			@include a4a-font-body-sm($font-weight: 600);
```

Comment:
The Figma design uses the Body MD with Font-weight 600 here. Can we confirm the correctness?

Reply:
Yes, this is for small screens as they are hard to fit in.